fj-extension/predefined-link.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Using Predefined Link
+description: "Learn how to use a predefined link to reproduce and capture logs with the Fiddler Jam extension easily."
+slug: fj-predefined-link
+publish: true
+position: 70
+---
+
+# Using Predefined Link
+
+This tutorial describes how to use Fiddler Jam predefined link.
+
+- First, you will install the lightweight Fiddler Jam Extension.
+
+- Next, you'll use the received Jam predefined link to record a network request.
+
+- Finally, you will submit the issue to the Fiddler Jam Portal cloud space, where the predefined link creator can further investigate your case.  
+
+
+## Recording a Log
+
+In this step, you'll record a captured traffic session so that you can isolate a network issue and then be able to share it with the engineering teams for further analysis.  
+
+1. Open the predefined Jam link shared with you by your collaborator (support agent, developer, etc.).
+
+    ![predefined link initial screen](../images/ext/prerecorded/prerecorded-000.png)
+
+
+1. Open the Fiddler Jam extension 
+
+    At this point, if you expand [**Advanced Options**]({%slug fj-capture-options%}), you'll see that some of the settings are enabled or disabled by default. Let's not change any of them for this guide and work with the default ones.
+
+1. Click **Start Capture** to start recording.
+
+    As a result, Fiddler Jam initiates the recording of all network activity in the current browser tab, and a notification appears informing you that Fiddler Jam has started debugging. Close the notification or click **Cancel** safely as these won't stop the capturing. Hide the Fiddler Jam window by clicking anywhere in the browser tab, and continue recording. Try not to exceed the maximum size of a recorded of 100MB.
+
+    ![Start capture](../images/ext/prerecorded/prerecorded-001.png)
+
+1. Enter the URL in the browser address bar. If the creator of the predefined link has set a default URL, it will be loaded
```

Comment:
```suggestion
1. Enter the URL in the browser address bar. If the creator of the predefined link has set a default URL, it will be loaded automatically.
```